engine/src/flutter/impeller/typographer/glyph_atlas.h

```diff
@@ -10,14 +10,7 @@
 #include <optional>
 
 #include "flutter/fml/build_config.h"
-
-#if defined(OS_FUCHSIA)
-// TODO(gaaclarke): Migrate to use absl. I couldn't get it working since absl
-// has special logic in its GN files for Fuchsia that I couldn't sort out.
-#define IMPELLER_TYPOGRAPHER_USE_STD_HASH
-#else
 #include "flutter/third_party/abseil-cpp/absl/container/flat_hash_map.h"
-#endif
```

Comment:
There's more stuff below that should be deleted (see where there is an `#if IMPELLER_TYPOGRAPHER_USE_STD_HASH`)

Reply:
I think you can also delete `AbslHashAdapter` and `AbslHashAdapterCombiner` since they are no longer used anymore.